### PR TITLE
9 part a adding test intensity normalisation transform

### PR DESCRIPTION
**Pull request by wyli**
- merge master branch's testing code into 9-intensity-normalisation-transform
- adding a unit test for intensity normalisation

### CODE REVIEW

File: monai/data/transforms/intensity_normalizer.py [hunk 1]
```diff
@@ -20,17 +23,19 @@ class IntensityNormalizer(Transform):
...
         divisor (ndarray): the amount to divide by (usually the standard deviation)
         dtype: output data format
     """
 
     def __init__(self, apply_keys, subtrahend=None, divisor=None, dtype=np.float32):
-        Transform.__init__(self)
-        assert apply_keys is not None and (type(apply_keys) == tuple or type(apply_keys) == list), \
-            'must set apply_keys for this transform.'
-        self.apply_keys = apply_keys
+        self.apply_keys = apply_keys if isinstance(apply_keys, (list, tuple)) else (apply_keys,)
```
Comment (Nic-Ma) [inline monai/data/transforms/intensity_normalizer.py]: I think you should set value after all below assert checks.

File: monai/data/transforms/transform.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
```
Comment (Nic-Ma) [inline monai/data/transforms/transform.py]: Why did you add additional 1 line? All other code files use 1 line blank between copyright and code.

File: tests/test_intensity_normalizer.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import unittest
```
Comment (Nic-Ma) [inline tests/test_intensity_normalizer.py]: need copyright.

File: tests/test_intensity_normalizer.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+    def test_image_normalizer_default(self):
+        data_key = 'image'
+        normalizer = IntensityNormalizer(data_key)  # test a single key
+        normalised = normalizer({data_key: self.imt})
+        expected = (self.imt - np.mean(self.imt)) / np.std(self.imt)
+        self.assertTrue(np.allclose(normalised[data_key], expected))
+
+    def test_image_normalizer_default_1(self):
+        data_key = 'image'
```
Comment (Nic-Ma) [inline tests/test_intensity_normalizer.py]: Why not test multiple keys as this test is for multiple keys?

File: monai/data/transforms/transform.py (hunk 2 above)
Comment (wyli) [reply monai/data/transforms/transform.py]: sure, I've updated the code. we might want to have some flake configurations to enforce this style.
